File: MyMovies/ClientApp/app/components/actors/actor.ts
export class Actor{
    id:number;
    firstName:string;
    lastName:string;
    birthday:Date;
    roles:Role[];
}

export class ActorInMovie{
    id:number;
    firstName:string;
    lastName:string;
    role:string;
}

export class Role{
    id:number;
    year:number;
    roleName:string;
    movieTitle:string;
    movieId:number;                                    
}

File: MyMovies/DAL/Actor.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace MyMovies.DAL
{
    public class Actor
    {
        public int Id { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        [DataType(DataType.Date)]
        public DateTime Birthday { get; set; }
        virtual public ICollection<Role> Roles { get; set; }

    }
}
